### PR TITLE
[docs] edit content for useResetRecoilState() page

### PR DESCRIPTION
**Pull request by irhamputra**
Hi 👋🏼
I created a sample code for useResetRecoilState()

### CODE REVIEW

File: docs/docs/api-reference/core/useResetRecoilState.md [hunk 1]
```diff
@@ -9,3 +9,12 @@ Returns a function that will reset the value of the given state to its default v
 - `state`: a writeable Recoil state
 
 ### Example
+
+```jsx
+import { todoListState } from "../atoms/todoListState";
```
Comment (drarmstr) [inline docs/docs/api-reference/core/useResetRecoilState.md:14]: To match our Prettier settings:
```jsx
import {todoListState} from "../atoms/todoListState";
```

File: docs/docs/api-reference/core/useResetRecoilState.md [hunk 2]
```diff
@@ -9,3 +9,12 @@ Returns a function that will reset the value of the given state to its default v
...
+
+const TodoReset = () => {
```
Comment (drarmstr) [inline docs/docs/api-reference/core/useResetRecoilState.md:16]:
```jsx
const TodoResetButton = () => {
```

File: docs/docs/api-reference/core/useResetRecoilState.md (hunk 1 above)
Comment (irhamputra) [reply docs/docs/api-reference/core/useResetRecoilState.md]: request changed

File: docs/docs/api-reference/core/useResetRecoilState.md (hunk 2 above)
Comment (irhamputra) [reply docs/docs/api-reference/core/useResetRecoilState.md]: request changed

**Pull request merged**